UKHomeOffice/cerberus-service
# COP-6432: Replace Momentjs with Dayjs

## Description
**Moment.js** is now a legacy project, and therefore we decided to update or code to make use of **Day.js** library to parse and manipulate dates, specially since **Day.js** uses the same **API** as moment.

## To Test
* clone repo;
* run project (native development or with docker)
* run all tests and ensure none breaks due to moment being replaced;
* quickly run the app in your localhost and check if the dates look accordingly.

## Developer Checklist

\* Required
